_integration-schemas/google-sheets/sheet_metadata.md

```diff
@@ -0,0 +1,85 @@
+---
+tap: "google-sheets"
+version: "1.0"
+key: "sheet_metadata"
+
+name: "sheet_metadata"
+doc-link: "https://developers.google.com/sheets/api/reference/rest"
+singer-schema: "https://github.com/singer-io/tap-google-sheets/blob/master/tap_google_sheets/schemas/sheet_metadata.json"
+description: "This table contains metadata about the sheets within a spreadsheet."
+
+replication-method: "Full Table"
+
+api-method:
+    name: "getSheets"
+    doc-link: "https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets/sheets#top_of_page"
+
+attributes:
+  - name: "columns"
+    type: "object"
+    description: ""
+    subattributes:
+      - name: "columnIndex"
+        type: "integer"
+        description: "The column index in the data table on which the filter is applied to."
+      - name: "columnLetter"
+        type: "string"
+        description: "The letter of the column."
+      - name: "columnName"
+        type: "string"
+        description: "The header name in the column."
+      - name: "columnType"
+        type: "string"
+        description: "The type of column"
+      - name: "columnSkipped"
+        type: "boolean"
+        description: "Field that identifies if the column has been skipped."
+      - name: "type"
+        type: "string"
+        description: ""
+      - name: "format"
+        type: "string"
+        description: ""            
+
+  - name: "gridProperties"
+    type: "object"
+    description: ""
```

Comment:
Add the description in here, since it's available: https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets/sheets#gridproperties

`The properties of a grid.`